README.md

```diff
@@ -52,14 +52,12 @@ Verify a successful upgrade by
 
 Changelog
 ---------
-* 0.38: *gluon 2016.2.4*
+* 0.38: *gluon 2017.1.3*
   * see
-    * https://gluon.readthedocs.io/en/v2016.2.2/releases/v2016.1.6.html
-    * https://gluon.readthedocs.io/en/v2016.2.2/releases/v2016.2.html
-    * https://gluon.readthedocs.io/en/v2016.2.2/releases/v2016.2.1.html
-    * https://gluon.readthedocs.io/en/v2016.2.2/releases/v2016.2.2.html
-    * https://gluon.readthedocs.io/en/v2016.2.4/releases/v2016.2.3.html
-    * https://gluon.readthedocs.io/en/v2016.2.4/releases/v2016.2.4.html
```

Comment:
Hier sollten die Release-Notes für alle Zwischenversionen drin bleiben, da davon auszugehen ist, dass alle Änderungen aus v2016.2.x auch in v2017.1.x enthalten sind, in den Releasenotes der späteren Version aber nicht nochmal separat aufgeführt werden.

Reply:
Dann Packen wir alle wieder rein